common/reviews/api/gulp-core-build-webpack.api.ts

```diff
@@ -9,7 +9,7 @@ interface IWebpackTaskConfig {
 // @public (undocumented)
 class WebpackTask extends GulpTask<IWebpackTaskConfig> {
   // (undocumented)
-  public executeTask(gulp: gulp.Gulp, completeCallback: (result?: Object) => void): void;
+  public executeTask(gulp: gulp.Gulp, completeCallback: (error?: string) => void): void;
```

Comment:
The API is actually currently wrong. It's not "result," and it'll throw if a non-string is passed in.

---
In reply to: [124898461](https://github.com/Microsoft/web-build-tools/pull/256#discussion_r124898461) [](ancestors = 124898461)